pywbem_mock/_wbemconnection_mock.py

```diff
@@ -109,94 +106,80 @@ def _uprint(dest, text):
-        repository (ex. cc = conn.GetClass(objectname)
-
-      objectname:
-        The object path of the target object, as follows:
-
-        * Instance-level call: The instance path of the
-          target instance, as a :class:`~pywbem.CIMInstanceName`
-          object including the execution namespace (either the
-          namespace defined by the client
-          InvokeMethod or the WBEMConnection default namespace if the
-          client InvokeMethod did not provide a namespace).
-
-        * Class-level call: The class path of the target
-          class, as a :class:`~pywbem.CIMClassName` object
-          including the execution namespace (either the
-          namespace defined by the client InvokeMethod or the
-          WBEMConnection default namespace if the
-          client InvokeMethod did not provide a namespace).
-
-          The :class:`~pywbem.CIMClassName` object,  `host`
-          attribute will be ignored.
-
-      methodname(:term:`string`):
-        The methodname from the InvokeMethod call. This allows
-        a single callback function to be used as the responder
-        for multiple methods.
-
-      params (:class:`Nocasedict`):
-        Input parameters aggregated from the InvokeMethod Params and params
-        input as a NocaseDict.
-
-        Each dictionary item represents a single input parameter
-        for the CIM method and is a :class:`~pywbem.CIMParameter` with.
-
-        * name (:term:`string`):
-          Parameter name (case independent)
-        * type (:term:`string`:
-          String representation of parameter type.
-        * value (:term:`CIM data type`):
-          Parameter value
+      conn (:class:`~pywbem_mock.FakedWBEMConnection`):
+        Faked connection. This can be used to access the mock repository of the
+        faked connection, via its operation methods (e.g. `GetClass`).
+
+      objectname (:class:`~pywbem.CIMInstanceName` or :class:`~pywbem.CIMClassName`):
+        The object path of the target object of the invoked method, as follows:
+
+        * Instance-level call: The instance path of the target instance, as a
+          :class:`~pywbem.CIMInstanceName` object which has its `namespace`
+          property set to the target namespace of the invocation. Its `host`
+          property will not be set.
+
+        * Class-level call: The class path of the target class, as a
+          :class:`~pywbem.CIMClassName` object which has its `namespace`
+          property set to the target namespace of the invocation. Its `host`
+          property will not be set.
+
+      methodname (:term:`string`):
+        The CIM method name that is being invoked. This is the method name for
+        which the callback function was registered. This parameter allows a
+        single callback function to be registered for multiple methods.
+
+      params (:ref:`NocaseDict`):
+        The input parameters for the method that were passed to the
```

Comment:
How about "the input parameters from the client InvokeMethod request.

Reply:
**Doscission point:** The input parameters of the InvokeMethod operation are not the same as the input parameters of the CIM method that is invoked.

Reply:
Agree. Ignore